### DISCUSSION TITLE
EarTrumpet shows OBS icons in all the output (Playback) audio devices

### DISCUSSION THREAD

**Discussion by RafflesParrot** [File-New-Project/EarTrumpet#1463] [Q&A] [answered]
*** Question

EarTrumpet shows OBS icons in all the output (Playback) audio devices.

*** Software

EarTrumpet 2.3.0.0
Windows 11 Pro, Version 22H2, OS Build 22621.2134
OBS Studio (29.1.3 64 bit)

*** Details

For OBS Studio (29.1.3 64 bit), EarTrumpet shows OBS icons in all the output (Playback) audio devices.

Audacity (3.3.3 64-bit) also has the same behavior.

I thought that a desktop app audio output could be associated with only a single output (Playback) audio device at a given time.

Why does this occur?

I am a software developer so you can go into the details.

I am heavily using OBS Studio and need to understand why OBS does this with EarTrumpet.

Thank you in advance for the feedback.

**Comment by riverar** [comment]
Can you:
* file an issue rather than a discussion @ https://github.com/File-New-Project/EarTrumpet/issues
* provide a screenshot

Thanks!

**Comment by RafflesParrot** [comment, marked as answer]
A recent BIOS update and NVIDIA update my have corrected this issue.

Moderator - Please delete this thread.
